src/zkevm_specs/tx.py

```diff
@@ -16,16 +16,15 @@ class Tag(IntEnum):
     Nonce = 1
     Gas = 2
     GasPrice = 3
-    GasTipCap = 4
-    GasFeeCap = 5
-    CallerAddress = 6
-    CalleeAddress = 7
-    IsCreate = 8
-    Value = 9
-    CallDataLength = 10
-    TxSignHash = 11
-    CallData = 12
-    Pad = 13
+    CallerAddress = 4
+    CalleeAddress = 5
+    IsCreate = 6
+    Value = 7
+    CallDataLength = 8
+    CallDataGasCost = 9
```

Comment:
sync `tables.md`? What's the difference between `Tag` and `TxContextFieldTag`?

Reply:
Ah good point... They are the same :sweat_smile:
I originally created a new type to account for small differences between the Tx Table seen by the EVM and the same defined by the Tx Circuit, but this PR unifies that, so I'll remove the `Tag` in favor of `TxContextFieldTag`

Reply:
Addressed in https://github.com/privacy-scaling-explorations/zkevm-specs/pull/243/commits/bdc1189fa4b923d78e0057bc05d4b6525f3c03a3